presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -151,3 +151,161 @@ Hive, this table can be described in Presto::
++------------------------------------------------+------------------------------------------+-------------------------------------------+--------------------------------------------------------------+
++      ``hive.orc.stream-`` ``buffer-size``      +          ``String`` (data size)          +                 ``8 MB``                  +*Unused*                                                      +
++------------------------------------------------+------------------------------------------+-------------------------------------------+--------------------------------------------------------------+
++ ``hive.parquet-optimized-`` ``reader.enabled`` +               ``Boolean``                +                 ``false``                 +*Deprecated* Enables number of reader improvements introduced +
++                                                +                                          +                                           +by alternative parquet implementation. The new reader         +
++                                                +                                          +                                           +supports vectorized reads, lazy loading, and predicate push   +
++                                                +                                          +                                           +down, all of which make the reader more efficient and         +
++                                                +                                          +                                           +typically reduces wall clock time for a query. However as the +
++                                                +                                          +                                           +code has changed significantly it may or may not introduce    +
++                                                +                                          +                                           +some minor issues, so it can be disabled if some problems     +
++                                                +                                          +                                           +with environment are noticed.                                 +
++------------------------------------------------+------------------------------------------+-------------------------------------------+--------------------------------------------------------------+
++``hive.parquet-predicate-`` ``pushdown.enabled``+               ``Boolean``                +                 ``false``                 +*Deprecated* Works as                                         +
++                                                +                                          +                                           +``hive.parquet-predicate-pushdown.enabled`` but allows to     +
```

Comment:
Did you mean: "Works as hive.parquet-optimized-reader.enabled ..." ?
In any case, the description is a bit confusing to me. How exactly those two properties interact with each other? Let's walk through all 4 combinations and clarify that.